packages/core/src/api/webhook.ts

```diff
@@ -0,0 +1,167 @@
+/* eslint-disable jsdoc/check-param-names */
```

Comment:
Is this needed?

Reply:
Yeah, it happens when a type literal is used for a type, jsdoc expects each nested field to be documented.